### PR TITLE
Fix oracle issue #1800

### PR DESCRIPTION
**Pull request by bigpear0201**
oracle查询返回columnName为大写，而snakeString方法返回值为小写，导致columnsMp[col]无法得到响应的值，这里将columnsMp构建时的key改成了小写。

### CODE REVIEW

**Comment by astaxie** [comment]
please gofmt

**Comment by bigpear0201** [comment]
change with gofmt

File: orm/orm_raw.go [hunk 1]
```diff
@@ -19,6 +19,7 @@ import (
 	"fmt"
 	"reflect"
 	"time"
+    "strings"
```
Comment (astaxie) [inline orm/orm_raw.go]: it looks still not align with other package

**Comment by miraclesu** [comment]
@bigpear0201 I think this is a wrong fix.
You can use `column` for special field. Doc for `column` usage: http://beego.me/docs/mvc/model/models.md#column

@astaxie close #1800 ?

**Pull request closed without merging**